src/Analyzers/CSharp/Tests/UseUtf8StringLiteral/UseUtf8StringLiteralTests.cs

```diff
@@ -761,50 +732,21 @@ public static void M1()
         }
 
         [Theory, Trait(Traits.Feature, Traits.Features.CodeActionsUseUtf8StringLiteral)]
-        // Various cases copied from https://github.com/dotnet/runtime/blob/main/src/libraries/Common/tests/Tests/System/Net/aspnetcore/Http3/QPackDecoderTest.cs
-        [InlineData(new byte[] { 0x37, 0x02, 0x74, 0x72, 0x61, 0x6e, 0x73, 0x6c, 0x61, 0x74, 0x65 }, "7translate")]
-        [InlineData(new byte[] { 0x3f, 0x01 }, "?")]
```

Comment:
Both of these inputs are now invalid, so moved the theory below, which then rendered this whole test unused